utils/utils.go

```diff
@@ -751,7 +752,13 @@ func IsValidEth1Tx(s string) bool {
 	return !zeroHashRE.MatchString(s) && eth1TxRE.MatchString(s)
 }
 
-// IsValidEth1Tx verifies whether a string represents a valid eth1-tx-hash.
+// IsEth1Tx verifies whether a string represents an eth1-tx-hash.
+// In contrast to IsValidEth1Tx, this also returns true for the 0x0 address
+func IsEth1Tx(s string) bool {
+	return eth1TxRE.MatchString(s)
+}
+
+// IsHash verifies whether a string represents an eth1-hash.
```

Comment:
+1 for documenting functions, wish we had this for db functions with explanations for the params 🙃